packages/p2p/src/components/buy-sell/buy-sell-row.jsx

```diff
@@ -15,13 +15,6 @@ import TradeBadge from '../trade-badge';
 const BuySellRow = ({ row: advert }) => {
     const { buy_sell_store, floating_rate_store, general_store } = useStores();
 
-    if (advert.id === 'WATCH_THIS_SPACE') {
-        // This allows for the sliding animation on the Buy/Sell toggle as it pushes
-        // an empty item with an item that holds the same height of the toggle container.
-        // Also see: buy-sell-table.jsx
-        return <div style={{ height: '140px' }} />;
-    }
-
```

Comment:
If we are removing these, please make sure to check the buy/sell table in p2p on mobile and confirm that everything works as expected.